core/src/i18n/en.yaml

```
@@ -1617,19 +1617,26 @@ stateful-sets:
   name_singular: Stateful Set
   replicas: Replicas
   title: Stateful Sets
-subscription:
+subscriptions:
   create:
     labels:
+      add-filter: Add filter
       application-name: Application name
       event-name: Event name
+      event-source: Event source
       event-type: Event type
       event-type-plural: Event types
       event-version: Event version
+      filter: Filter {{name}}
+      filters: Filters
       service-name: Service
       sink: Sink
+    messages:
+      one-filter-required: At least one filter is required.
     placeholders:
       application-name: Choose an Application name
       event-name: Enter the Event name, for example, order.cancelled
+      event-source: Enter the Event source
```

```suggestion
      event-source: Enter the event source
```

Events are capitalized only when the meaning behind it is the [Kubernetes resource](https://v1-19.docs.kubernetes.io/docs/tasks/debug-application-cluster/events-stackdriver/), not our regular Eventing events. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms) for reference. 

Please correct the whole file (and any other if needed).